pkg/kando/process_client.go

```diff
@@ -43,3 +55,45 @@ func processAsJSONFlagValue(cmd *cobra.Command) bool {
 	}
 	return b
 }
+
+func proxySetup(ctx context.Context, addr string, pid int64) {
```

Comment:
Can we use the `proxySetup()` from your previous PR that already merged here: https://github.com/kanisterio/kanister/pull/3335/files#diff-2ed1938fd31979cf263c8dfc679327d8641260d70da102c57053e8785693c48bR56

This function seems refactored but is basically identical to the the one preexisting in `pkg/kando/process_client_output.go` inside master branch.

Maybe also rebase your branch from master ?

Reply:
Oh nvm, looks like you moved this here from `pkg/kando/process_client_output.go` in this PR.